src/openapi/streaming/connection/transport/websocket-transport.ts

```diff
@@ -377,6 +391,43 @@ class WebsocketTransport implements StreamingTransportInterface {
         }
     }
 
+    private startInactivityFinder = () => {

```

Comment:
is there a chance of start being called when it is already running?

Reply:
startInactivityFinder is attached to socket.onopen listener and it will be invoked only once when we create the stocket, i have added logging to track this further